package.json

```diff
@@ -25,14 +25,15 @@
   "author": "Atanas Korchev",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/akorchev/odata2openapi/issues"
+    "url": "https://github.com/elasticio/odata2openapi/issues"
   },
-  "homepage": "https://github.com/akorchev/odata2openapi#readme",
+  "homepage": "https://github.com/elasticio/odata2openapi#readme",
   "devDependencies": {
     "@types/node": "^7.0.18",
+    "ts-node": "^8.1.0",
     "typescript": "^2.3.2"
   },
   "dependencies": {
-    "xml2js": "^0.4.17"
+    "xml-js": "^1.6.11"
```

Comment:
In general, we avoid `^` in our pacakge dependencies.

Reply:
Fixed